File: src/minecraft/monnef/jaffas/trees/item/ItemTrees.java
/*
 * Jaffas and more!
 * author: monnef
 */

package monnef.jaffas.trees.item;

import monnef.jaffas.food.item.ItemJaffaBase;
import monnef.jaffas.trees.JaffasTrees;
import monnef.jaffas.trees.common.Reference;

public class ItemTrees extends ItemJaffaBase {
    public ItemTrees(int v) {
        super(v);
        setCreativeTab(JaffasTrees.instance.CreativeTab);
    }

    @Override
    public String getModName() {
        return Reference.ModName;
    }

    @Override
    public int getDefaultSheetNumber() {
        return 2;
    }
}

File: src/main/scala/monnef/jaffas/trees/item/ItemTrees.scala
package monnef.jaffas.trees.item

import monnef.jaffas.food.item.ItemJaffaBase
import monnef.jaffas.trees.JaffasTrees
import monnef.jaffas.trees.common.IconDescriptorTrees

class ItemTrees(_id: Int, _texture: Int) extends ItemJaffaBase(_id, _texture) with IconDescriptorTrees {
  def this(v: Int) {
    this(v, 0)
    setCreativeTab(JaffasTrees.instance.creativeTab)
  }
}


